README.md

```diff
@@ -424,6 +424,93 @@ The local key helper is responsible for:
 - Producing signatures with the binding key
 - Cleanup of the binding key and its artifacts (when the user clears the browser session or the key is unused for a long time)
 
+### Local Key Helper on Android
+On android platform a Local Key Helper is a protocol interface (`ILocalKeyHelper.java`), and its implementation (`LocalKeyHelperImpl.java`) is yet to be determined.
+
+Local KeyHelpers on the Android platform are implemented using [content providers](https://developer.android.com/guide/topics/providers/content-providers). These content providers allow browser applications, such as Chrome, to interact with them by querying and accessing specific APIs defined by the Local KeyHelpers.
+
+#### Content Provider Authority
+Each Local KeyHelper defines a content provider with a authority such as `com.contoso.localkeyhelper`. Different APIs are implemented as separate paths on this content provider.
```

Comment:
Is authority a commonly known concept in content provider? I'm not very familiar

Reply:
Yes, Authority is a symbolic name for the ContentProvider and is part of the Content URI when communicating with a contentprovider
https://developer.android.com/guide/topics/providers/content-provider-basics#ContentURIs